examples/configs/test_example.yml

```diff
@@ -0,0 +1,73 @@
+# lightning.pytorch==2.0.4
+seed_everything: true
+trainer:
+  accelerator: auto
+  strategy: auto
+  devices: auto
+  num_nodes: 1
+  precision: 32-true
+  callbacks: null
+  fast_dev_run: false
+  max_epochs: null
+  min_epochs: null
+  max_steps: -1
+  min_steps: null
+  max_time: null
+  limit_train_batches: null
+  limit_val_batches: null
+  limit_test_batches: null
+  limit_predict_batches: null
+  overfit_batches: 0.0
+  val_check_interval: null
+  check_val_every_n_epoch: 1
+  num_sanity_val_steps: null
+  log_every_n_steps: null
+  enable_checkpointing: null
+  enable_progress_bar: null
+  enable_model_summary: null
+  accumulate_grad_batches: 1
+  gradient_clip_val: null
+  gradient_clip_algorithm: null
+  deterministic: null
+  benchmark: null
+  inference_mode: true
+  use_distributed_sampler: true
+  profiler: null
+  detect_anomaly: false
+  barebones: false
+  plugins: null
+  sync_batchnorm: false
+  reload_dataloaders_every_n_epochs: 0
+  default_root_dir: null
+model:
+  architecture: null
+  model_config: {}
+  loss_function: null
+  lr: 0.001
+  schedule: Constant
+  log_num_samples: 8
+  test_cellpose_model_path: null
+  test_cellpose_diameter: null
+  test_evaluate_cellpose: false
+data:
+  data_path: null
+  source_channel: null
+  target_channel: null
+  z_window_size: null
+  split_ratio: 0.8
+  batch_size: 16
+  num_workers: 8
+  yx_patch_size:
+  - 256
+  - 256
+  augment: true
+  caching: false
+  normalize_source: false
+  ground_truth_masks: null
+  train_z_scale_range:
+  - 0.0
+  - 0.0
+  train_noise_std: 0.0
```

Comment:
I thought this were obtained from the HCSDataModule.setup('fit')(?)

Reply:
It infers the correct shape based on the user input here:

https://github.com/mehta-lab/VisCy/blob/3f46b3b8e210c478ecceb7b1ec05618c2abb045f/viscy/light/data.py#L440-L448

Can you explain more?

Reply:
Sorry, I had the wrong intuition on what these were supposed to do. Since I didn't know we could run the config through the CLI.